lib/proto.js

```diff
@@ -193,7 +193,7 @@ function write (_obj, msg, num) {
     }
   } else {
     obj = _obj
-    if (msg === undefined && _obj[errorKey]) {
+    if (msg === undefined && _obj.msg === undefined && _obj[errorKey]) {
```

Comment:
Hi all, should this be `_obj[messageKey]` (instead of `_obj.msg`) since that `msg` key is configurable?

Reply:
Done: https://github.com/pinojs/pino/pull/1746. I did two separate `git push`es to try and trigger the failing test on CI but it didn't work as approval is needed. Hopefully you'll be able to trigger CI on the first commit!